pkg/kv/kvnemesis/kvnemesis_test.go

```diff
@@ -35,33 +42,124 @@ func init() {
 	numSteps = envutil.EnvOrDefaultInt("COCKROACH_KVNEMESIS_STEPS", 50)
 }
 
-func TestKVNemesisSingleNode(t *testing.T) {
-	defer leaktest.AfterTest(t)()
-	defer log.Scope(t).Close(t)
-	skip.UnderRace(t)
+func testClusterArgs(tr *SeqTracker) base.TestClusterArgs {
+	storeKnobs := &kvserver.StoreTestingKnobs{
+		// Drop the clock MaxOffset to reduce commit-wait time for
+		// transactions that write to global_read ranges.
+		MaxOffset: 10 * time.Millisecond,
+		// Make sure we know the seq for each of our writes when they come out of
+		// the rangefeed. We do this via an interceptor to avoid having to change
+		// RangeFeed's APIs.
+		RangefeedValueHeaderFilter: func(key, endKey roachpb.Key, ts hlc.Timestamp, vh enginepb.MVCCValueHeader) {
+			seq := kvnemesisutil.Seq(vh.Seq)
+			if false && len(endKey) > 0 {
+				fmt.Printf("XXX rangefeed intercept ranged write [%s,%s), seq %s\n", key, endKey, seq)
+			}
+			if seq > 0 {
+				tr.Add(key, endKey, ts, seq)
+			}
+		},
```

Comment:
Kind of bikesheddy, but I'm not super-thrilled about this approach. Would it be too bad to add a default-off option to include value headers in rangefeeds too?

Reply:
MVCCValueHeader isn't plumbed through rangefeeds yet, I'd prefer that from a simplicity point of view but is it maybe taking it too far? We'd have to add them [here](https://github.com/cockroachdb/cockroach/blob/c57b340ef74e6e4d4c64fb9e318a690518d49eed/pkg/roachpb/api.proto#L2715-L2724), I think.

Reply:
Yes, and for `RangeFeedDeleteRange` too. I think I'd prefer to just plumb them through, but let's get @nvanbenschoten's take on all of this too.